src/compiler/test-file/formats/es-next/compiler.js

```diff
@@ -54,13 +55,28 @@ export default class ESNextTestFileCompiler extends APIBasedTestFileCompilerBase
     }
 
     _getRequireCompilers () {
-        return {
+        const requireCompilers = {
```

Comment:
Put the used strings (`.js`, `.jsx`, `.cjs`, `.mjs`) into constants or enum and use them here and later.